src/signalrclient/CMakeLists.txt

```diff
@@ -61,6 +60,14 @@ else()
     target_compile_options(signalrclient PRIVATE -Wextra -Wpedantic -Wno-unknown-pragmas)
 endif()
 
+if(INCLUDE_JSONCPP)
+  target_sources(signalrclient PRIVATE ../../third_party_code/jsoncpp/jsoncpp.cpp)
+  target_include_directories(signalrclient PRIVATE ../../third_party_code/jsoncpp)
+else()
+  target_link_libraries(signalrclient PUBLIC ${JSONCPP_LIB})
+  unset(JSONCPP_LIB CACHE)
```

Comment:
What is this for?

Reply:
Hold-over from some testing I was doing. Thanks for catching